From 433fdc9f7514784750b62a1a9d3d6a746cdf6a7c Mon Sep 17 00:00:00 2001
From: lch <lchopn@gmail.com>
Date: Wed, 1 Apr 2026 13:59:55 +0800
Subject: [PATCH 2/3] Fix about version number for packaging

---
 ui/src/dialogs/AboutDialog.jsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/src/dialogs/AboutDialog.jsx b/ui/src/dialogs/AboutDialog.jsx
index 486cc049..c70a72af 100644
--- a/ui/src/dialogs/AboutDialog.jsx
+++ b/ui/src/dialogs/AboutDialog.jsx
@@ -91,7 +91,7 @@ const LinkToVersion = ({ version }) => {
     ? `https://github.com/navidrome/navidrome/compare/v${
         parts[0].split('-')[0]
       }...${commitID}`
-    : `https://github.com/navidrome/navidrome/releases/tag/v${parts[0]}`
+    : `https://github.com/navidrome/navidrome/releases/tag/v${parts[0].split('-')[0]}`
   return (
     <>
       <Link href={url} target="_blank" rel="noopener noreferrer">
-- 
2.53.0

