From c22c831dd5acf720ca3d23dcfd6b55826a8225f6 Mon Sep 17 00:00:00 2001
From: lch <lchopn@gmail.com>
Date: Wed, 1 Apr 2026 14:00:19 +0800
Subject: [PATCH 3/3] Update dependency to specify taglib 2

---
 adapters/taglib/taglib_wrapper.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/adapters/taglib/taglib_wrapper.go b/adapters/taglib/taglib_wrapper.go
index 4a979920..85b12da3 100644
--- a/adapters/taglib/taglib_wrapper.go
+++ b/adapters/taglib/taglib_wrapper.go
@@ -1,7 +1,7 @@
 package taglib
 
 /*
-#cgo !windows pkg-config: --define-prefix taglib
+#cgo !windows pkg-config: --define-prefix taglib-2
 #cgo windows pkg-config: taglib
 #cgo illumos LDFLAGS: -lstdc++ -lsendfile
 #cgo linux darwin CXXFLAGS: -std=c++11
-- 
2.53.0

